.github/workflows/ci.yml

```diff
@@ -18,42 +18,83 @@ jobs:
           - elixir: 1.4.x
             otp: 20
             os: ubuntu-20.04
```

Comment:
This runner image has been retired and can't be used by a GitHub Actions workflow job any more. I'll see if I can find a replacement image.
```
This is a scheduled Ubuntu 20.04 retirement. Ubuntu 20.04 LTS runner will be removed on 2025-04-15. For more details, see https://github.com/actions/runner-images/issues/11101
```

Reply:
I ended up resolving this problem by creating a similar job in [71212eb](https://github.com/michalmuskala/jason/pull/200/commits/71212ebffed14493b3ec662ef6ef82666a952bec) and slicing the parts of the build matrix which depend on Ubuntu 20.04 in a container instead. It will make the workflow definition a little more cumbersome to maintain but the older Elixir & OTP pairs are unlikely to change.